### ISSUE TITLE
Making System analysis module generic

### ISSUE THREAD

**Issue by Muktan** [01-vyom/End_2_End_Automatic_Speech_Recognition_For_Gujarati#23]
`system_analysis.py` can be made generic so that other people can use it to perform their system analysis in the Gujarati language.

Currently, the problem with it is that it takes a static CSV file name and the column names are also hardcoded.

**Comment by Muktan** [comment]
I will be working on this issue.

**Comment by 01-vyom** [comment]
Project discussion needs to be done to approve it.

**Comment by Muktan** [comment]
Yes definitely, I will start working once we complete the project discussion.
